example/lib/main.dart

```diff
@@ -0,0 +1,649 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/rx_form.dart';
+import 'package:widget_toolkit/widget_toolkit.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Widget Toolkit Demo',
+      theme: ThemeData.light()
+          .copyWith(colorScheme: ColorScheme.fromSwatch(), extensions: [
+        WidgetToolkitTheme.light,
+        ItemPickerTheme.light,
+        SearchPickerTheme.light,
+        TextFieldDialogTheme.light,
+        EditAddressTheme.light,
+      ]),
+      darkTheme: ThemeData.dark()
+          .copyWith(colorScheme: ColorScheme.fromSwatch(), extensions: [
+        WidgetToolkitTheme.dark,
+        ItemPickerTheme.dark,
+        SearchPickerTheme.dark,
+        TextFieldDialogTheme.dark,
+        EditAddressTheme.dark,
+      ]),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key});
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  late PageController pageController;
```

Comment:
You can initialise the page controller as follows, which will make the `initState` redundant
`late PageController pageController = PageController();`